README.md

```diff
@@ -166,6 +166,13 @@ Cohesion and Reuse in an Object-Oriented System,<br/>
 Department of Computer Science, Colorado State University, 1995,
 [PDF](http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.53.2683).
 
+[`hitz95`]
+Lack of Cohesion in Methods (**LCOM4**).<br/>
```

Comment:
@llorllale "Lack of Cohesion in Methods 4 (**LCOM4**)"

Reply:
@amihaiemil ok